docs/deployment/provisionviaiapp.md

```diff
@@ -0,0 +1,7 @@
+# Provisioning VIAI Edge app
```

Comment:
Just noticed this one. Is this document something to elaborate?

Reply:
This is a residue from the original PDF instructions. The content is already covered in the other pages. Deleting.